app/lib/cms/s3.rb

```diff
@@ -6,6 +6,8 @@ module S3
 
     FIPS_FILE_PATH = "/proc/sys/crypto/fips_enabled"
 
+    DEFAULT_EXPIRES_IN = 900
```

Comment:
actually, since we control this for the logo. And logo is not critical at all, I'd make the validity of the logo URL a day or so

Reply:
I was trying to think what this number affects... and I actually couldn't figure out.
I believe the image will only be accessed through the URL when the pages is reloaded or when invoice is regenerated, so URL is generated and accessed within the same request, so seconds at most. Why would we need a longer expiration?

Reply:
Because of why not. It can in fact be public so why not give more headroom To avoid potential inconveniences.